src/pages/academy/search/_article-card.tsx

```diff
@@ -41,9 +38,30 @@ const ArticleCard = ({ items }: ArticleCardProps) => {
 
     return (
         <Flex mb="40px" jc="flex-start" tablet={{ mb: '24px' }}>
-            <IconWrapper src={ArticleIcon} alt="article icon" />
+            <Desktop>
+                <StandardImgWrapper width="161px" height="96px" border_radius="6px">
+                    <QueryImage
+                        data={items.main_image.imageFile}
+                        alt={items.main_image.description || ''}
+                        className="standard-query-img"
+                    />
+                </StandardImgWrapper>
+            </Desktop>
             <Flex max-width="auto" ml="14px" fd="column">
-                <Flex jc="space-between" tablet={{ fd: 'column', jc: 'flex-start' }}>
+                <Mobile>
+                    <StandardImgWrapper
+                        tabletL_width="120px"
+                        tabletL_height="72px"
+                        tabletL_border_radius="4px"
```

Comment:
Can we use either `tablet` or `tabletS` instead of `tabletL`.. The design looks weird when the breakpoint starts at `tabletL`.

Reply:
Very good point @njazuli-deriv please check new realization 🙏